csrc/scheduler/pointwise.cpp

```diff
@@ -29,37 +29,6 @@ namespace {
 // Unused at the moment, commenting for clang tidy
 constexpr int64_t kThreadX = 128;
 
-class DomainMap : public pointwise_utils::DomainMap {
```

Comment:
This part is moved to pointwise_utils.h so that it can be also used from the resize scheduler